git-extra/vimrc

```diff
@@ -25,6 +25,12 @@ set statusline+=%=              " Rest: right align
 set statusline+=%l,%c%V         " Position in buffer: linenumber, column, virtual column
 set statusline+=\ %P            " Position in buffer: Percentage
 
+if &term =~ 'xterm'
+  let &t_ti.="\e[1 q"           " Use a blinking block cursor when entering termcap mode
```

Comment:
Does this let the cursor blink by default?

I wonder how that interacts with MinTTY. I configured a block cursor there, and I could imagine that other users did so, too. Yet other users may have configured a line cursor specifically and could possibly be pretty annoyed by this here patch.

What's your take on this?